src/provider/controller/NotificationCenter.ts

```diff
@@ -73,6 +73,13 @@ export class NotificationCenter extends AsyncInit {
         this._trayIcon.onLeftClick.add(() => {
             new ToggleCenterVisibility(ToggleCenterVisibilitySource.TRAY).dispatch(this._store);
         });
+        try {
+            await fin.GlobalHotkey.register('ctrl+alt+n', async () => {
```

Comment:
We probably want to use `commandorcontrol` instead of just `ctrl` to make it work properly on Mac too (e.g. as we did in [layouts](https://github.com/HadoukenIO/layouts-service/blob/3ad170039e9f2432a2e1d1b2170e81b05b9ad366/src/provider/snapanddock/SnapService.ts#L51))

Reply:
Are you going to add this?

Reply:
@tomrobertsOF fixed this.